packages/node_modules/@webex/plugin-meetings/src/util/webex-errors.js

```diff
@@ -0,0 +1,95 @@
+// The document would auto generate the doc for errors
```

Comment:
We already have errors in the `common/errors` folder, could we use that instead?

Reply:
fixed